server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java

```diff
@@ -62,34 +71,37 @@
     private static final String PASSWORD = "secret";
     private static final String COUNT = "count";
     private static final String SIZE = "size";
-    private MaxQuotaManager maxQuotaManager;
-    private UserQuotaRootResolver userQuotaRootResolver;
-    private InMemoryCurrentQuotaManager currentQuotaManager;
-
-    @BeforeEach
-    void setUp(RestQuotaSearchTestSystem testSystem) throws Exception {
-        DomainList domainList = testSystem.getQuotaSearchTestSystem().getDomainList();
-        domainList.addDomain(Domain.of(PERDU_COM));
-        domainList.addDomain(Domain.of(STRANGE_ORG));
-
-        UsersRepository usersRepository = testSystem.getQuotaSearchTestSystem().getUsersRepository();
-        usersRepository.addUser(BOB.asString(), PASSWORD);
-        usersRepository.addUser(JACK.asString(), PASSWORD);
-        usersRepository.addUser(THE_GUY_WITH_STRANGE_DOMAIN.asString(), PASSWORD);
-
-        RestAssured.requestSpecification = testSystem.getRequestSpecification();
-        RestAssured.enableLoggingOfRequestAndResponseIfValidationFails();
-
-        maxQuotaManager = testSystem.getQuotaSearchTestSystem().getMaxQuotaManager();
-        userQuotaRootResolver = testSystem.getQuotaSearchTestSystem().getQuotaRootResolver();
-        currentQuotaManager = testSystem.getQuotaSearchTestSystem().getCurrentQuotaManager();
+
+    interface SetUp {
```

Comment:
you are not supposed to do that, you should use nesting to share things, you can very well use non-static test classes with @Nested